website/components/CommunityCompatibleTable.tsx

```diff
@@ -107,6 +112,23 @@ export const CommunityPluginCompatibleTable: React.FC = () => {
       url: 'https://github.com/waysact/webpack-subresource-integrity',
       status: CompatibleStatus.NotCompatible,
     },
+    {
+      name: 'directory-named-webpack-plugin',
+      url: 'https://github.com/shaketbaby/directory-named-webpack-plugin',
+      status: CompatibleStatus.NotCompatible,
+      description: i18n[lang]['resolve-plugin-un-support-desc'],
+    },
+    {
+      name: 'pnp-webpack-plugin',
+      url: 'https://github.com/arcanis/pnp-webpack-plugin',
+      status: CompatibleStatus.NotCompatible,
+      description: i18n[lang]['resolve-plugin-un-support-desc'],
+    },
+    {
+      name: 'webpack-virtual-modules',
+      url: 'https://github.com/sysgears/webpack-virtual-modules',
+      status: CompatibleStatus.NotCompatible,
```

Comment:
can we use https://github.com/rspack-contrib/rspack-plugins/tree/main/packages/plugin-virtual-module